SearchResults.js

```diff
@@ -53,6 +53,9 @@ function SearchResults() {
   self.agesOfPRsOpenAt = function (when) {
     return self.openPRsAtDate(when).map(function (pr) { return when - pr.createdAt; });
   };
+  self.lastPRUpdate = function () {
+    return _.max(self.pullRequests().map(function (pr) { return pr.updatedAt; }));
```

Comment:
As you are pulling in update order, you should be able to just check the last update of the last PR, there's no need to iterate through all of them.